pass-model/src/main/java/org/dataconservancy/pass/model/Policy.java

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright 2018 Johns Hopkins University
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.dataconservancy.pass.model;
+
+import java.net.URI;
+
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Describes a Policy. Policies determine the rules that need to be followed by a Submission.
+ * @author Karen Hanson
+ * @version $Id$
+ */
+
+public class Policy extends PassEntity {
+
+    @JsonProperty("@type")
+    protected String type = PassEntityType.POLICY.getName();
+    
+    /** Title of policy e.g. "NIH Public Access Policy" */
+    private String title;
+    
+    /** Several sentence description of policy */
+    private String description;
+    
+    /** List of URIs for repositories that can satisfying this policy */
+    private List<URI> repositories;

```

Comment:
Specifically, for example, the list of repositories that can satisfy a given policy should probably be looked up in real time by a policy engine at some point in the future.